python/tvm/relay/qnn/op/qnn.py

```diff
@@ -280,6 +281,9 @@ def conv2d(data,
         The computed result.
     """
 
+    # TODO enforce 4-way padding in topi/nn/conv2d after #4644 merged
+    # convert 2-way padding to 4-way padding
```

Comment:
Why this is related with #4644?

Reply:
This is to make Relay Conv2d and QNN conv2d handle 4D padding in same manner. Relay Conv2d looks like this

https://github.com/apache/incubator-tvm/blob/master/python/tvm/relay/op/nn/nn.py#L204-L206

I guess QNN changes got missed earlier.

Reply:
Ok. Got it.